openlibrary/templates/books/works-show.html

```diff
@@ -3,7 +3,8 @@
 <span itemscope itemtype="https://schema.org/Book">
   <span class="bookcover">
     $ url = book.get_cover_url("S") or "/images/icons/avatar_book-sm.png"
-    <a href="$book.key"><img itemprop="image" src="$url" height="60"/></a>
+    $ urlB= book.get_cover_url("M") or "/images/icons/avatar_book-lg.png"
+    <a href="$book.key"><img itemprop="image" src="$url" srcset= "$url,$urlB 2x" height="60"/></a>
```

Comment:
I don't think you need to report $url in srcset. I think "$urlB 2x" should be plenty here.